### PR TITLE
add the tutorial for IsoTools

### PR DESCRIPTION
**Pull request by YalanBi**
Here is the tutorial for analysing SG-NEx data using IsoTools. Please have a review and let us know if there is any issue. Many thanks!

### CODE REVIEW

File: docs/SG_NEx_IsoTools_tutorial.ipynb [hunk 1]
```diff
@@ -0,0 +1,1408 @@
...
...JYUb09u3b0f//v3x8OFDNG7cGI8ePYK3tzeCgoIAAI0aNcKlS5ewZMkSLFy4EO3atcPt27eVDkg046Snp6fUNi8vLwBQaOZHkdjYWKxevRoBAQHYt28f3rx5o3RMjDHGWGkJCAjA7NmzUatWLUyfPh21a9fG7NmzsWLFCgCAhYUF9u/fj99//x0HDx5Eo0aNcOrUKTVHzRhjjFU+CnfzbtiwIapVq4bo6GhoaGhAKBSiU6dOSElJwZ07dyTyJiUlYcqUKYiIiEBOTo5SAQ0cOBAHDx7E9evX0bJlS6nthoaGMDU1RVJSUoHlyJuATE9PD35+fvjmm28K3D8nJ0ci9vT0dFhZWXG3CMZYpcBdwdSnfv36aNCgAY4dOyZO69mzJ+7du4f4+HiJvJ9OCJqXl1faoVYI/F1njFUWfL1TPYWfTCcnJ6Ndu3bQ0Pi4i4aGBr744gskJydL5a1bty6OHTuG4OBgpQNKS0sD8LFbtyxGRkbiPAWxtbXF+vXrcf/+fbx//x7JycnYsWMHzMzM8O2332L9+vUF7r9s2TIYGxuLXxVpmS/GGGNl1+vXr+Hk5CSR5ujoiNevX0vlFU0Ieu7cudIKjzHGGGP/T+En0+3atcOrV68QHR2N2rVr4+nTp3B1dYW5uTkuX76ssoA8PT1x+vRpxMXFyVz+o3bt2sjIyFCoQS3L7du30apVK1StWhUvXryApqbsCc35yTRjrDLju9fq06VLF9y6dQthYWFo0aIFYmNj0bdvXzRu3BinT59Wd3gVDn/XGWOVBV/vVE/hJ9OrVq3C8+fPYWNjg1q1asHGxgZPnz7FqlWrVBqQ6Im0vMay6EtQVI0aNUKHDh2QkpKCf//9V24+HR0dGBkZSbwYY4yxkrZ+/XpoaGigffv20NPTwxdffAGBQIBffvlF3aExxhhj7BMKrzPdvn17/Pvvv9i1axeSk5NhZWWFYcOGoW7duioNyN7eHgAQFxcnNWb6+fPnyMjIgIuLS7GOYW5uDgDIzMwsVjmMMcaYqjk5OeH+/fs4evSouL7t2bMnDA0N1R0aY4wxxj6hcGMaAKysrDB//vySigUA4OrqimXLluHUqVMYMmSIxLbw8HBxnqLKz8/H9evXAaDQtaoZY4wxdTAwMMDQoUPVHQZjjDHGCqDUOtOlwcPDA/Xq1UNoaChu3LghTk9LS8PSpUuhra2N
```
Comment (cying111) [inline docs/SG_NEx_IsoTools_tutorial.ipynb:1367]: code line 1367 to 1373 creates an empty code chunk, can delete it?

Comment (cying111) [inline docs/SG_NEx_IsoTools_tutorial.ipynb:1366]: related to the comment below, the comma can also be deleted here

**Review by cying111 — APPROVED** [review]
Thanks for submitting the tutorial! The tutorial is very comprehensive and detailed, with nice illustrations and I can also run it without any difficulty. Only one minor suggestion related to the empty chunk in the end.

**Comment by YalanBi** [comment]
Thank you @cying111. The empty code chunk has been removed. Please help and merge the PR if there's no other issue. Many thanks in advance.

**Pull request merged**